include/config.inc.php

```diff
@@ -13,18 +13,21 @@
 /* Time zone to use for news items etc. */
 date_default_timezone_set("UTC");
 
+$proto = (isset($_SERVER["HTTPS"]) && $_SERVER["HTTPS"]) == "on" ? "https": "http";
+
 /* Base URL to the website. */
 if ($_SERVER['SERVER_PORT'] == '80') {
-	$url = "http://{$_SERVER['SERVER_NAME']}" . dirname($_SERVER['PHP_SELF']);
+	$url = "{$proto}://{$_SERVER['SERVER_NAME']}" . dirname($_SERVER['PHP_SELF']);
```

Comment:
Don't actually need to include a proto. If it's blank it will just use whatever the page was loaded with.

Reply:
`baseurl` is derived from the `BASE_URL` constant defined in this section.
RSS feeds also use `baseurl`. I have no idea what `//residualvm.org` will resolve to, there.

To be honest I'd prefer to make the site HTTPS-only, like ScummVM.

Reply:
does it solve it? https://test.residualvm.org

Reply:
website already has redirections to https. but main page images seems are not redirected somehow

Reply:
I didn't notice before port in url. remove it, fixed redirection

Reply:
btw. you pushed your branch on this repo.